teachertool/src/components/HomeScreen.tsx

```diff
@@ -53,6 +58,60 @@ const Card: React.FC<CardProps> = ({ title, className, icon, onClick }) => {
     );
 };
 
+interface LoadingCardProps {
+    delay?: boolean;
+}
+
+const LoadingCard: React.FC<LoadingCardProps> = ({ delay }) => {
+    return (
+        <div className={css.cardContainer}>
+            <Button
+                className={classList(css.cardButton, css.loadingGradient, delay ? css.loadingGradientDelay : undefined)}
+                title={""}
+                onClick={() => {}}
+            >
+                <div className={css.cardDiv}>
+                    <div className={css.loadingGradient}></div>
+                </div>
+            </Button>
+        </div>
+    );
+};
+
+interface RubricResourceCardProps {
+    cardTitle: string;
+    imageUrl: string;
+    rubricUrl: string;
+}
+
+const RubricResourceCard: React.FC<RubricResourceCardProps> = ({ cardTitle, imageUrl, rubricUrl }) => {
+    const onRubricClickedAsync = async () => {
+        pxt.tickEvent(Ticks.LoadRubric, { rubricUrl });
+        await loadRubricAsync(rubricUrl);
+    };
+    return (
+        <div className={css.cardContainer}>
+            <Button
+                className={classList(css.cardButton, css.rubricResource)}
+                title={cardTitle}
+                onClick={onRubricClickedAsync}
+            >
+                <div
+                    className={classList(css.cardDiv)}
+                    style={{
+                        backgroundImage: `url("${window.location.origin}${imageUrl}")`,
```

Comment:
Is there a way to point at the right image without using `window.location.origin`?